committee.html

```diff
@@ -130,10 +130,10 @@ <h1 class="darkcolor">THE COMMITTEE</h1>
                             <div class="card shadow-sm h-100">
                               <svg class="bd-placeholder-img card-img-top" viewBox="0 0 100 100" role="img" preserveAspectRatio="xMidYMid meet" focusable="false"><image href="img/team/sarah.jpeg" width="100%" height="100%"></svg>
                               <div class="card-body">
-                                <p class="card-text custom-font bold-font darkcolor" style="font-size:160%;">Sarah Truong</p>
-                                <p class="text-muted custom-font darkcolor">Earth and Planetary Science, Computer Science, & Data Science</p>
+                                <p class="card-text custom-font bold-font darkcolor" style="font-size:160%; padding-top: 10px !important;">Sarah Truong</p>
```

Comment:
Can you delete the "&"? It looks weird, and Sarah's the only one with three majors 😳, so we don't need to worry about consistency.